src/updateDisplay.js

```diff
@@ -0,0 +1,37 @@
+import pressedAnotherOperator from "./pressedAnotherOperator";
```

Comment:
@grimxreaper in the commit "extract updateDisplay, reduce onClick setState calls, add isDisplayab...", how could you have refactored changeKeys to become updateDisplay so that you aren't deleting changeKeys and adding updateDisplay as a new file?

Reply:
I would have also passed in `lastEquation` to changeKeys and then added the additional `if` conditionals.